test-data/message/mixed-up-long.eml

```diff
@@ -9,10 +6,6 @@ User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:102.0) Gecko/20100101
 Content-Language: en-US
 To: bob@example.net
 From: Alice <alice@example.org>
-X-Mozilla-Draft-Info: internal/draft; vcard=0; receipt=0; DSN=0; uuencode=0;
- attachmentreminder=0; deliveryformat=0
-X-Identity-Key: id3
-Fcc: imap://alice%40example.org@in.example.org/Sent
```

Comment:
This is a real-world message, so, why remove these headers? I mean, sure, they don't change anything today, but still, I think it's good to simulate the real world as closely as possible and just keep the email as it was, rather than going through it and removing all headers that are ignored by DC today.